src/main/groovy/com/blackducksoftware/integration/hub/detect/bomtool/pear/PearDependencyFinder.groovy

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (C) 2017 Black Duck Software Inc.
+ * http://www.blackducksoftware.com/
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Black Duck Software ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Black Duck Software.
+ */
+package com.blackducksoftware.integration.hub.detect.bomtool.pear
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+import com.blackducksoftware.integration.hub.bdio.simple.model.DependencyNode
+import com.blackducksoftware.integration.hub.bdio.simple.model.Forge
+import com.blackducksoftware.integration.hub.bdio.simple.model.externalid.NameVersionExternalId
+import com.blackducksoftware.integration.hub.detect.util.DetectFileManager
+import com.blackducksoftware.integration.hub.detect.util.executable.Executable
+import com.blackducksoftware.integration.hub.detect.util.executable.ExecutableOutput
+import com.blackducksoftware.integration.hub.detect.util.executable.ExecutableRunner
+
+@Component
+class PearDependencyFinder {
+    private final Logger logger = LoggerFactory.getLogger(PearDependencyFinder.class)
+    private final Forge pear = new Forge('pear', '/')
+
+    @Autowired
+    DetectFileManager detectFileManager
+
+    @Autowired
+    ExecutableRunner executableRunner
+
```

Comment:
If you have this return a List<DependencyNode> of children, you can do the parsing of the xml file in the bom tool itself to create the DetectCodeLocation with the more structured constructor